notebooks/en/rag_zephyr_langchain.ipynb

```diff
@@ -155,9 +155,9 @@
       },
```

Comment:
<p>Let's also update this paragraph then. Suggestion:</p><p> </p><p>"The content of individual GitHub issues may be longer than what an embedding model can take as input. If we want to embed all of the available content, we need to chunk the documents into appropriately sized pieces.</p><p> </p><p>The most common and straightforward approach to chunking is to define a fixed size of chunks and whether there should be any overlap between them. Keeping some overlap between chunks allows us to preserve some semantic context between the chunks. The recommended splitter for generic text is the <a href="https://python.langchain.com/docs/modules/data_connection/document_transformers/recursive_text_splitter" target="_blank">RecursiveCharacterTextSplitter</a> , and that's what we'll use here. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Thanks for the suggestion! I have committed the same.